include/lsst/afw/math/TransformBoundedField.h

```diff
@@ -0,0 +1,94 @@
+// -*- LSST-C++ -*-
+/*
+ * LSST Data Management System
+ * Copyright 2017 LSST Corporation.
```

Comment:
It starts out as the year of authorship. That is changed to a range for future edits in a later year.